gcp/modules/gcp-stackdriver-monitoring/alert_flowmanager_uptime.tf

```diff
@@ -7,7 +7,7 @@ resource "google_monitoring_alert_policy" "flowmanager_uptime" {
       filter          = "metric.type=\"monitoring.googleapis.com/uptime_check/check_passed\" AND resource.label.host=\"flowmanager.${var.domain_name}\" AND resource.type=\"uptime_url\""
       comparison      = "COMPARISON_GT"
       threshold_value = 1.0
-      duration        = "300s"
+      duration        = "${(var.env == "prd" || var.env == "stg") ? "300" : "600"}s"
```

Comment:
What difference between environments prevents us from using same condition duration? Is there a specific scenario that triggers false positives for ephemeral envs?

Reply:
Yes, in the case of ephemeral envs, the alert is triggered at the creation of the cluster and the workloads. It seems that the alert is triggered before the service is up.